docs/source/user_guide/installation.md

```diff
@@ -1,79 +1,113 @@
 (target-installation)=
 # Installation
 
-## Install the package
-:::{admonition} Use a conda environment
-:class: note
-To avoid dependency conflicts with other packages, it is best practice to install Python packages within a virtual environment.
-We recommend using [conda](conda:) or [mamba](mamba:) to create and manage this environment, as they simplify the installation process.
-The following instructions assume that you have conda installed, but the same commands will also work with `mamba`/`micromamba`.
-:::
+## Use a virtual environment
 
-### Users
-To install movement in a new environment, follow one of the options below.
-We will use `movement-env` as the environment name, but you can choose any name you prefer.
+While not strictly required, it is highly recommended to install `movement` into a
+clean virtual environment, using tools such as [conda](conda:) or [uv](uv:).
+
+This should be set up before installing `movement`.
 
 ::::{tab-set}
-:::{tab-item} Conda
-To create an environment with the core package only:
+:::{tab-item} conda
+Create and activate a new environment:
 ```sh
-conda create -n movement-env -c conda-forge movement
+conda create -y -n movement-env -c conda-forge python=3.13
+conda activate movement-env
 ```
 
-If you wish to use the GUI, which additionally requires [napari](napari:),
-you should instead run:
+We used `movement-env` as the environment name, but you can choose any name you prefer.
+:::
+
+:::{tab-item} uv
+Make sure [uv is installed on your system](uv:getting-started/installation).
+Then create a [virtual environment](uv:pip/environments/) in your project directory:
+
 ```sh
-conda create -n movement-env -c conda-forge movement napari pyqt
+uv init
```

Comment:
why do we need this`?
```suggestion
```

Reply:
Thanks for flagging this. I initially thought that this would be required for projects without an existing `pyproject.toml` file (e.g. many research data analysis projects), but turns out that's not the case. 

`uv venv` and `uv pip install` will work without uv initialisation.

Reply:
In short, `uv init` would be needed only if you are setting a new project from scratch, but I don't think we need to spell it out here, uv users will deduce as much from the uv docs. I think I will just scrap `uv init` from our instructions.